agent/src/main/java/org/openremote/agent/protocol/dmx/AbstractArtnetLight.java

```diff
@@ -0,0 +1,82 @@
+package org.openremote.agent.protocol.dmx;
+
+public abstract class AbstractArtnetLight {
```

Comment:
This is not an abstract class...there's nothing abstract in it....combine with `ArtnetLight`